### ISSUE TITLE
[Feature Request]: Emulate joycons with the console's gyroscope to use with something like the Hori Split Pad Pro?

### ISSUE THREAD

**Issue by jonahnm** [ndeadly/MissionControl#522]
### Feature Description

The feature would allow people in handheld mode whose joycons don't have a gyroscope to be able to use motion controls in a variety of games.

### Rationale

This feature would be very helpful to me and plenty of people who bought the Hori Split Pad Pro, to realize later it had no motion controls.

**Comment by ndeadly** [comment]
Interesting idea. I don't even know how to access the console motion data, so will have to look into that first.

**Comment by monstrosities** [comment]
Great idea, if it's actually possible this would be incredible.

**Comment by zcufiq** [comment]
> Interesting idea. I don't even know how to access the console motion data, so will have to look into that first.

Any updates? I cant complete some of the shrines in BOTW without motion controls and I only have these HORI Split Pad Compacts that don't include motion controls.

**Comment by ndeadly** [comment]
No. I asked around and it seems like in order to access the raw motion data I need to talk to the hardware directly over SPI. This is more trouble than I have time for at the moment. There are higher priority tasks to take care of.

**Comment by zcufiq** [comment]
> No. I asked around and it seems like in order to access the raw motion data I need to talk to the hardware directly over SPI. This is more trouble than I have time for at the moment. There are higher priority tasks to take care of.

oh ok. thanks for answering

**Comment by imsofi** [comment]
If its of any help, i know Captain Toad has support for the internal gyroscope: https://www.reddit.com/r/NintendoSwitch/comments/fy0mbm/so_i_found_out_in_some_games_gyro_works_from/

This is especially frustrating, since following the new release of the Hori Split Pad Compact, it was shipped without Gyro's as well. Following in line with most other officially licensed joy-cons. :/

**Comment by ndeadly** [comment]
The issue here is about how to talk to the device at the hardware level. This is all abstracted away by the sdk when games interact with it. It would need to be researched/reverse engineered and implemented for multiple hardware revisions, which is a non-trivial amount of work.
